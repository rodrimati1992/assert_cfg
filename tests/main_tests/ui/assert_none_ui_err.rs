assert_cfg::none!{
    feature = "foo",
    feature = "bar",
    feature = "baz",
}

assert_cfg::none!{
    feature = "foo",
    feature = "bar",
    feature = "qux",
}

assert_cfg::none!{
    feature = "foo",
    feature = "world",
}


fn main(){}