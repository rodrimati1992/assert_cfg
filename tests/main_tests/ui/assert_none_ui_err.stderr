error[E0080]: evaluation of constant value failed
 --> tests/main_tests/ui/assert_none_ui_err.rs:1:1
  |
1 | / assert_cfg::none!{
2 | |     feature = "foo",
3 | |     feature = "bar",
4 | |     feature = "baz",
5 | | }
  | |_^ the evaluated program panicked at '
these features must be disabled:
- `feature = "foo"`
- `feature = "bar"`
- `feature = "baz"`
', $DIR/tests/main_tests/ui/assert_none_ui_err.rs:1:1
  |
  = note: this error originates in the macro `assert_cfg::none` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0080]: evaluation of constant value failed
  --> tests/main_tests/ui/assert_none_ui_err.rs:7:1
   |
7  | / assert_cfg::none!{
8  | |     feature = "foo",
9  | |     feature = "bar",
10 | |     feature = "qux",
11 | | }
   | |_^ the evaluated program panicked at '
these features must be disabled:
- `feature = "foo"`
- `feature = "bar"`
', $DIR/tests/main_tests/ui/assert_none_ui_err.rs:7:1
   |
   = note: this error originates in the macro `assert_cfg::none` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0080]: evaluation of constant value failed
  --> tests/main_tests/ui/assert_none_ui_err.rs:13:1
   |
13 | / assert_cfg::none!{
14 | |     feature = "foo",
15 | |     feature = "world",
16 | | }
   | |_^ the evaluated program panicked at '
these features must be disabled:
- `feature = "foo"`
', $DIR/tests/main_tests/ui/assert_none_ui_err.rs:13:1
   |
   = note: this error originates in the macro `assert_cfg::none` (in Nightly builds, run with -Z macro-backtrace for more info)
